dart/test/sentry_envelope_vm_test.dart

```diff
@@ -50,248 +53,21 @@ void main() {
       final sdkVersion = SdkVersion(name: '', version: '');
       final attachment =
           IoSentryAttachment.fromPath('this_path_does_not_exist.txt');
-
       final envelope = SentryEnvelope.fromEvent(
         event,
         sdkVersion,
+        dsn: fakeDsn,
         attachments: [attachment],
       );
 
       final data = (await envelope.envelopeStream(SentryOptions()).toList())
           .reduce((a, b) => a + b);
 
-      expect(data, envelopeBinaryData);
+      final file = File('test_resources/envelope-no-attachment.envelope');
+      final jsonStr = await file.readAsString();
+      final dataStr = utf8.decode(data);
```

Comment:
This seems pretty hacky, but maybe it solves the failure?
```suggestion
      final file = File('test_resources/envelope-no-attachment.envelope');
      final jsonStr = await file.readAsString();
      final dataStr = utf8.decode(data.replaceAll('\r\n', '\n'));
```

Reply:
yeah I did that locally, but I dont like it.